oc-cluster

```diff
@@ -61,6 +73,13 @@ OS_DEFAULT_PROJECT="myproject"
 #
 __RESTART=0
 
+function which-ip {
+  if [[ "$__PLATFORM" == "linux" ]]; then
+    /sbin/ifconfig $1 | grep "inet " | awk -F: '{print $1}' | awk '{print $2}'
```

Comment:
I'll add it but should be the first one. Why do you have 4 ips for docker0? Anyone would work?